src/angular/src/swiper.component.ts

```diff
@@ -807,7 +809,14 @@ export class SwiperComponent implements OnInit {
         this.swiperRef.params[_key] = defaultParams;
       }
     }
-
+    if (_key === 'enabled') {
+      if (value === true) {
+        this.swiperRef.enable();
+      } else {
```

Comment:
I think it should be `} else if (value === false) {` on this line, otherwise I guess it will disable it if it is `undefined` (not passed)

Reply:
Done :)